public/sw.js

```diff
@@ -23,15 +23,27 @@ const fetchEventHandler = async event => {
     if (req.headers.has('accept')) {
       supportsWebp = req.headers.get('accept').includes('webp');
     }
```

Comment:
Consider using the `supportsWebp` inference below since the comparison below will set
supportsWebp to `true` or `false` without the need to set a default value via
```
let supportsWebp = false;
```

```suggestion
    const supportsWebp = req.headers.has('accept') && req.headers.get('accept').includes('webp');
```

Reply:
Is this the logic that determines whether the OS supports webp? (Or do we need to lookup the User-Agent?)

Also, ideally we want to default supportsWebp to true

Reply:
Yes, the comment already in the code when I started said '// Inspect the accept header for WebP support' above this bit of code. You might have written it a while ago actually haha! I am trying to check the headers on the versions of iOS that don't work, but SauceLabs is being very slow to start the dev tools

Reply:
Cool thanks - yes it has been a while since I looked at this code, so memory is foggy! 

To make life a bit easier for you, why don't you use the preview environment for Saucelabs - the preview pipeline is quite clever in that it will kick off a new release when you push commits to the PR. You can continue working on fixing the tests, but testing the behaviour on an environment which is more representative of a test environment (with service worker working properly)